docs/UserGuide.md

```diff
@@ -81,10 +70,61 @@ Throughout the guide, you'll encounter various icons that signify different type
+* **Manage orders and points**: Seamlessly add orders which automatically updates customers' points to reflect customer purchases and interactions.
+* **Update membership tiers**: Automatically elevate members' tiers based on their membership points, unlocking new rewards and benefits aligned with each tier.
+
+## 2.1 Understanding Membership Tiers and Points System
+SweetRewards features a dual-point system designed to enhance customer engagement and loyalty: `points` and `membership points`.
+
+### Points vs. Membership Points
+SweetRewards employs a nuanced points system to foster both immediate and long-term customer loyalty. Below is a comparative overview of the two types of points within the program, showcasing their distinct roles:
+
+| Attribute | `Points`                                                             | `Membership points`                                                   |
+|-----------|----------------------------------------------------------------------|-----------------------------------------------------------------------|
+| Purpose   | Serve as currency within the loyalty program for immediate rewards   | Track long-term customer engagement and determine membership tier     |
+| Earning   | Earned through purchases and specific interactions within the bakery | Earned identically to Points but are cumulative and never decrease    |
+| Redeeming | Can be redeemed for discounts, offers, or products                   | Not redeemable; solely influence tier status                          |
+| Impact    | Affect immediate customer benefits and rewards                       | Determine the member's tier and long-term benefits within the program |
+| Commands  | `addorder` increases points; `redeempoints` decreases points         | Only increased by `addorder`; unaffected by `redeempoints`            |
+
+🌟 **Benefit**: While both types of points are earned through purchases, `Points` cater to immediate gratification by being redeemable for rewards. In contrast, `Membership Points` are designed to recognize and reward the cumulative loyalty of customers, influencing their progression through the membership tiers without direct redemption for goods or services.
+
+### Membership Tiers: Criteria and Benefits
+Members are classified into tiers based on their accumulated membership points. Each tier unlocks distinct benefits, rewarding members for their loyalty:
+
+* Bronze Tier:
+  * Entry-level for all new members. 
+  * Offers basic rewards and incentives. 
+
+* Silver Tier:
+  * Requires members to accumulate a specified threshold of membership points. 
+  * Unlocks enhanced benefits and rewards.
+  
+* Gold Tier:
+  * A higher tier requiring more membership points. 
+  * Offers substantial rewards and exclusive offers. 
+
+* Platinum Tier:
+  * The pinnacle of the membership hierarchy. 
+  * Members enjoy the best rewards and personalized services. 
```

Comment:
I think can just put a table with the tier name and points required. Maybe also want to put a tip that the name, number, and point requirement of Tiers cannot be changed for now, and is still in development. Users should adjust the number of points their items give in order to suit their use